src/components/FilterAnnotModal/FilterAnnotModal.js

```diff
@@ -299,12 +300,12 @@ const FilterAnnotModal = ({ coAssessors, notesShareTypesMap, shareTypeColors })
       <div className="filter">
         <div className="heading">{t('option.filterAnnotModal.shareType')}</div>
         <div className="buttons" style={{ gridTemplateColumns: `114px 100px` }}>
-          {[...statuses].map((val, index) => {
+          {[...Object.keys(ShareTypes)].map((val, index) => {
```

Comment:
Maybe the choices could be moved to another render function? Just for cleanup purposes

Reply:
To keep changes as low as possible, I'll leave this as it is.